src/tests/specs/smoke/smokeTest.spec.ts

```diff
@@ -50,6 +50,12 @@ describe('Planner Smoke Tests:', () => {
     expect(await planner.workItemList.hasWorkItem(c.updatedWorkItem.title)).toBeTruthy();
   });
 
+  it('Check WorkItem title is not empty', async () => {
```

Comment:
The test is about if you can save it when the title is empty. I'd re-work on the test message.

Reply:
Nice job adding a test.

Reply:
Done https://github.com/fabric8-ui/fabric8-planner/pull/2491/commits/0e0fd06766d06a02981ef5b3370b37a50b5ff6cf